src/LibraryPage.tsx

```diff
@@ -200,6 +233,13 @@ const LibraryPage: React.FC = () => {
             }
           }}
         ></BookEditButton>
+        <LibraryDeleteButton
+          onDelete={(b) => {
+            if (b) {
+              deleteBook(b);
```

Comment:
Instead of creating a function here, it may be cleaner to do `onDelete={deleteBook}` and handle the `if` statement inside the `deleteBook` function